fatal error when building  on Windows "fatal error C1017"

https://github.com/isl-org/Open3D/blob/d6db89582328fb75e60a3da4f952dcef0d1987eb/cpp/open3d/visualization/visualizer/GuiVisualizer.cpp#L681


D:\Open3D\cpp\open3d\visualization\visualizer\GuiVisualizer.cpp(681): fatal error C1017: invalid integer constant expression [D:\Open3D\build\cpp\open3d\visualization\visualization.vcxproj]

I think it should be` #ifdef WIN32`

--- a/cpp/open3d/visualization/visualizer/GuiVisualizer.cpp
+++ b/cpp/open3d/visualization/visualizer/GuiVisualizer.cpp
@@ -678,7 +678,7 @@ void GuiVisualizer::Init() {
         file_menu->AddItem("Open...", FILE_OPEN, gui::KEY_O);
         file_menu->AddItem("Export Current Image...", FILE_EXPORT_RGB);
         file_menu->AddSeparator();
-#if WIN32
+#if defined(WIN32)
         file_menu->AddItem("Exit", FILE_QUIT);
 #elif !defined(__APPLE__)  // quit goes in app menu on macOS
         file_menu->AddItem("Quit", FILE_QUIT, gui::KEY_Q);

